console-consumer-primitive-keys-values/README.md

```diff
@@ -0,0 +1,61 @@
+# Consuming non-string primitive keys and values 
+
+You want to inspect or debug records written to a topic. Each record key and value is a long and double, respectively. In this tutorial, you'll learn how to specify key and value deserializers with the console consumer.
+
+
+## Setup
+
+To consume records from the command-line, cd into the <KAFKA BASE DIR>/bin directory. Then execute the following command (assuming keys with a type of `Long` and values of `Double`).
```

Comment:
suggest adding a producer command to this section so that readers can more easily complete the circle. makes it more easily runnable

Reply:
I left the producer part out intentionally because it only supports string keys and values at the moment 😭

Reply:
ah ok to leave it as is... I see that the original tutorial uses datagen connector to get around that. 😭  indeed